### PR TITLE
Disable RMRK1/2 Minting

### LINKED ISSUE #9387: Disable minting on RMRK/KSM
### Is your feature request related to a problem?

Currently minting on RMRK/KSM is no longer viable option as we have AssetHubs where fees are already very friendly and with RFC they would become even lower.

### Describe the solution you would like

I would like to see disabled minting on RMRK/KSM so users can choose only AHP and AHk

### Describe alternatives you have considered

People can already migrate their NFTs from RMRK/KSM to AHK/AHP

### Screenshots

_No response_

### Are there opened related issues?

- https://github.com/kodadot/ops-internal/discussions/1327

### PR DESCRIPTION
**Pull request by prury**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

maybe not the best solution, let me know

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #9387 

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13s37n7Y9VvoF1JcqnQzbL2b76qb9vNQQyXvpvpProdrpvBe)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

image

image

### CODE REVIEW

File: components/massmint/Massmint.vue [hunk 1]
```diff
@@ -11,6 +11,7 @@
             <p class="mb-4">{{ $t('massmint.chooseCollection') }}</p>
             <ChooseCollectionDropdown
               :preselected="preselectedCollectionId"
+              :disabled="isRemark"
```
[the remark this answers is not in the document]
Comment (prury) [reply components/massmint/Massmint.vue:14]: true, but how is it working? maybe it takes this property from the other components inside it? 
can you explain to noob me?

**Comment by exezbcz** [comment]
looks good to me, thanks for considering the warning if users lands with rmrk/2 selected

File: components/massmint/Massmint.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/massmint/Massmint.vue]: > true, but how is it working? maybe it takes this property from the other components inside it?
can you explain to noob me?
To enable it, we could add a new prop inside `ChooseCollectionDropdown ` component.

**Comment by prury** [comment]
@Jarsen136 ty, done

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.34 USD/DOT ~ 3.597 $DOT
🧗 [13s37n7Y9VvoF1JcqnQzbL2b76qb9vNQQyXvpvpProdrpvBe](https://polkadot.subscan.io/account/13s37n7Y9VvoF1JcqnQzbL2b76qb9vNQQyXvpvpProdrpvBe)
🔗 [0xa0f71912e177741699ec968f4415084fc595bff8c29cf76dde60a032b4183724](https://polkadot.subscan.io/extrinsic/0xa0f71912e177741699ec968f4415084fc595bff8c29cf76dde60a032b4183724)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**